src/log_surgeon/finite_automata/PrefixTree.cpp

```diff
@@ -0,0 +1,18 @@
+#include "PrefixTree.hpp"
+
+namespace log_surgeon::finite_automata {
+[[nodiscard]] auto PrefixTree::get_reversed_positions(id_t const index
```

Comment:
```suggestion
auto PrefixTree::get_reversed_positions(id_t const index
```
Attribute is only needed for function declaration

Reply:
Whoops must have been left in from when the function was moved out of the header. Fixed.